### DISCUSSION TITLE
ECHIPA WEB-DEVELOPERI

### DISCUSSION THREAD

**Discussion by sebiboga** [peviitor-ro/community#119] [TEAMs]
Calin Catuta
role: Front-end Developer
discord: Calin

**Comment by sebiboga** [comment]
Diana Dascalu
role: Front-end Developer
discord:  Dascalu Elena

**Comment by CalinCatuta** [comment]
Dehelean Catalin
role: Full-stack Developer
discord: Dehelean Catalin

**Comment by CalinCatuta** [comment]
Laurentiu Baluta
role: Full-stack Developer
discord: Laurentiu

**Comment by CalinCatuta** [comment]
Robert Burcea
role: Front-end Developer
discord: Robert Burcea

**Comment by AdrianaCionca** [comment]
Avem nevoie de un team lead si un tester

**Comment by CalinCatuta** [comment]
Role: UI
discord : Patricia
